apps/src/blockly/addons/cdoXml.js

```diff
@@ -88,7 +88,7 @@ export default function initializeBlocklyXml(blocklyWrapper) {
      * https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach
      */
     Array.prototype.forEach.call(xml.childNodes, function(xmlChild) {
-      if (xmlChild.tagName.toLowerCase() !== 'block') {
+      if (xmlChild.nodeName.toLowerCase() !== 'block') {
```

Comment:
Turns out `tagName` is only defined for element nodes ([stack overflow post](https://stackoverflow.com/questions/4878484/difference-between-tagname-and-nodename))
This also matches the logic we have in our fork already: https://github.com/code-dot-org/blockly/blob/main/core/utils/xml.js#L314